src/components/MyPage/MyInfo/index.tsx

```diff
@@ -1,13 +1,20 @@
+import { IcChangeProfileBtn } from "../../../asset/icon/index";
 import useUserProfile from "../../../core/api/myPage";
 import { St } from "./style";
+
 export default function MyInfo() {
   const { userProfile, isLoading, isError } = useUserProfile();
 
   return (
     <St.MyInfoContainer>
       {userProfile && (
         <St.Profile>
-          <St.ProfileImage src={userProfile.data.profile_image_url} alt="프로필이미지" />
+          <St.Images>
+            <St.ProfileImage src={userProfile.data.profile_image_url} alt="프로필이미지" />
+            <St.ChangeButton>

```

Comment:
시멘틱 태그는 스크린리더에 태그명도 같이 말하게 돼!
그래서 img 태그는 "이미지"라는 언급도 되기 떄문에 alt 값에 "이미지"를 넣어주지 않는 게 더 자연스러워요
프로필이미지 이미지 이렇게 나올 거기 떄문에!?

```suggestion
          <St.Images>
            <St.ProfileImage src={userProfile.data.profile_image_url} alt="프로필" />
            <St.ChangeButton>

```